server/src/js/Audit.js

```diff
@@ -95,7 +96,8 @@ class Audit {
       //assert(req.method, req.method);
       //assert(req.user);
       //assert(req.user.id);
-      operator = req.user.id;
+      operator = JSON.parse(req.headers.user)
```

Comment:
I doubt this is the correct way to do so, @luacmartins your idea is to pass in the user info directly from client right? I think you don't need to do so, and this might also be unsafe. 
The idea is: parse the auth token from client, get the user info, inject into the req, and get it in the audit middleware, then put into the DB